### PR TITLE
removed file checker from e-mail of application recieved

### PR DESCRIPTION
**Pull request by thesuss**
lets see how many fails ^_-

### CODE REVIEW

File: app/views/shf_application_mailer/acknowledge_received.html.haml [hunk 1]
```diff
@@ -12,10 +12,4 @@
       = shf_app_to_html(@shf_application)
   %hr
 
-  - if @shf_application.uploaded_files.size == 0
```
Comment (weedySeaDragon) [inline app/views/shf_application_mailer/acknowledge_received.html.haml:15]: Need to also change the specification `spec/mailers/shf_application_mailer_spec.rb`, which is the failing RSpec.
In that file, remove these lines:  

`lines 131 - 134`

```
   it 'no files uploaded' do
        expect(email_sent).to have_body_text(I18n.t('mailers.shf_application_mailer.acknowledge_received.review_after_recevied_files'))
      end
```

**Review by weedySeaDragon — CHANGES_REQUESTED** [review]
Merge with the current `develop` branch and fix the specification per my comment.

File: app/views/shf_application_mailer/acknowledge_received.html.haml (hunk 1 above)
Comment (thesuss) [reply app/views/shf_application_mailer/acknowledge_received.html.haml:15]: awesome comment. Dunno how I missed it for 20 days :o

File: spec/mailers/shf_application_mailer_spec.rb [hunk 2]
```diff
@@ -125,26 +125,6 @@ def make_branding_payment(a_company)
...
-      it 'a file uploaded (text not shown)' do
-        fn = file_fixture('image.gif')
-        received_app.uploaded_files.create(actual_file: File.open(fn, 'r'))
-        expect(email_sent).not_to have_body_text(I18n.t('mailers.shf_application_mailer.acknowledge_received.review_after_recevied_files'))
-        received_app.uploaded_files.delete_all
-      end
-
-    end
-
-    it_behaves_like 'it shows how to login and the page to upload files' do
```
Comment (weedySeaDragon) [inline spec/mailers/shf_application_mailer_spec.rb:144]: This test should stay [lines 144 - 146].  Don't delete it.  (This text is still in the email; we need to verify that it is there.)

**Review by weedySeaDragon — CHANGES_REQUESTED** [review]

**Review by weedySeaDragon — APPROVED** [review]

**Comment by weedySeaDragon** [comment]
Failing cucumber feature is one that we know intermittently fails, so it's fine to merge.

**Pull request merged**